apps/condo/domains/common/utils/session.js

```diff
@@ -0,0 +1,75 @@
+const get = require('lodash/get')
+const has = require('lodash/has')
+const uniq = require('lodash/uniq')
+
+const POSSIBLE_SESSION_PATHS = [
+    'req.session',
+    'session',
+]
+
+function _getSession (sessionOrWrapper) {
+    for (const sessionPath of POSSIBLE_SESSION_PATHS) {
+        if (has(sessionOrWrapper, sessionPath)) {
+            return get(sessionOrWrapper, sessionPath)
+        }
+    }
+
+    return sessionOrWrapper
```

Comment:
Why do you return `sessionOrWrapper ` here? 

It must be null, not passed argument